gammapy/modeling/models/spatial.py

```diff
@@ -45,6 +52,36 @@ def position(self):
         except IndexError:
             raise ValueError("Model does not have a defined center position")
 
+    _position_error = None
+
+    @property
+    def position_error(self):
+        """Set position error from covariance matrix"""
+        if self.parameters.covariance is None:
+            return self._position_error
+        pars = self.parameters
+        idx = [pars._get_idx("lon_0"), pars._get_idx("lat_0")]
+        sub_covar = pars.covariance[idx, :][:, idx]
+        cos_lat = self.lat_0.quantity.to_value("rad")
```

Comment:
Put the `np.cos` call on this line, to make the variable name `cos_lat` true, and then remove the `np.cos` call on the three next lines.